pkg/machine/ignition/ignition.go

```diff
@@ -743,6 +776,37 @@ func (i *IgnitionBuilder) Build() error {
 	return i.dynamicIgnition.Write()
 }
 
+func GetNetRecoveryFile() string {
+	return `#!/bin/bash
+# Verify network health, and bounce the network device if host connectivity
+# is lost. This is a temporary workaround for a known rare qemu/virtio issue
+# that affects some systems
+
+sleep 120 # allow time for network setup on initial boot
+while true; do
+  sleep 30
+  curl -s -o /dev/null --connect-timeout 10 http://192.168.127.1/health
```

Comment:
Where does this IP Address come from?

Reply:
This is the gvproxy gateway address used by the guest. In addition to routing gvproxy runs a built in http server for management of port forwards on this address:

https://github.com/containers/gvisor-tap-vsock/blob/8912b782e96b60da1455bf711eb620d893affa4a/cmd/gvproxy/main.go#L51